package/MDAnalysis/analysis/density.py

```diff
@@ -827,4 +865,4 @@ def __repr__(self):
             grid_type = 'density'
         else:
             grid_type = 'histogram'
-        return '<Density ' + grid_type + ' with ' + str(self.grid.shape) + ' bins>'
+        return f"<Density {grid_type} with {self.grid.shape} bins>"
```

Comment:
Did you test this line?

Reply:
No, perhaps I'll have to think about a test for this. I'd leave it alone, but it looks like autoPEP8 felt that the line was very long!

Reply:
Hi @orbeckst, I've removed this change. However, a check seems to be failing and I'm not sure if it is a concern.(?)

Reply:
The test failure was down to a non-related RDKIT test (it's a flaky test that fails sometimes on CI), I've restarted the jobs, hopefully they should pass this time.